src/borg/archiver.py

```diff
@@ -2155,6 +2155,13 @@ def do_break_lock(self, args, repository):
             A root path starts with the prefix `R`, followed by a path (a plain path, not a
             file pattern). An include rule starts with the prefix +, an exclude rule starts
             with the prefix -, an exclude-norecurse rule starts with !, all followed by a pattern.
+
+            .. note::
+
+                Via ``--pattern`` or ``--patterns-from`` you can define BOTH inclusion and exclusion
+                of files using pattern prefixes ``+`` and ``-``. With ``--exclude`` and
+                ``--exlude-file`` ONLY excludes are defined.
```

Comment:
it is not `--exlude-file`, but `--exclude-from`.

one could also continue the last sentence with "... and you need to use ``--include`` ... to define includes ONLY.

Reply:
Stupid me...

But I'm not sure what you mean with --include. There is no --include mentioned in e.g. borg create, is this a new undocumented option?